src/components/Card/HashCard/index.tsx

```diff
@@ -74,36 +74,36 @@ export default ({
 
   return (
     <HashCardPanel isColumn={!!iconUri}>
-      <div className="hash__card__content__panel" id="hash_content">
+      <div className="hashCardContentPanel" id="hash_content">
         {iconUri && isMobile ? (
           <div>
-            <img className="hash__icon" src={iconUri} alt="hash icon" />
-            <div className="hash__title">{title}</div>
+            <img className="hashIcon" src={iconUri} alt="hash icon" />
+            <div className="hashTitle">{title}</div>
           </div>
         ) : (
           <>
-            {iconUri && <img className="hash__icon" src={iconUri} alt="hash icon" />}
-            <div className="hash__title">{title}</div>
+            {iconUri && <img className="hashIcon" src={iconUri} alt="hash icon" />}
+            <div className="hashTitle">{title}</div>
           </>
         )}
 
         <div className={styles.hashCardHeaderRight}>
-          <div className="hash__card__hash__content">
+          <div className="hashCardHashContent">
             {loading ? (
               <LoadingPanel>
                 <SmallLoading />
               </LoadingPanel>
             ) : (
-              <div id="hash__text">
+              <div id="hashText">
```

Comment:
`id` should not be included in this update. And in some cases (not this one), an update of `id` impacts functionality, e.g., anchors on the page.